.gitignore

```
@@ -138,6 +138,7 @@ docs/misc/licenses.md
 docs/misc/dependencies.md
 QGISDeploymentToolbelt_*Ubuntu*
 tests/fixtures/tmp/*
+tests/fixtures/tmp_*
```

Almost duplicated line. Why is necessary?

```suggestion
tests/fixtures/tmp/*
```